### PR TITLE
Project ops improvement

### PR DESCRIPTION
**Pull request by froi**
# Why

We were having issues managing deployments to staging and production. There was no clear way to manage these. It was decided to improve our use of CircleCI by adding deploy steps to the config for both staging and production. An integration step was also added.

# What changed

- Our linter and integrations tests were not being used during our builds. This was added to our CircleCI workflow.
- The CircleCI script was updated to use version 2.0 and implement workflows.
- Linter errors were fixed.
- Integration tests were added to our CircleCI config.
- The package.json was cleaned up and contributors were added.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -35,3 +54,75 @@ jobs:
...
+          name: Install Dependencies
+          command: npm install
+      - save_cache:
+          key: code-gov-web-{{ checksum "package.json" }}
+          paths:
+            - code-gov-web/node_modules
+      - run:
+          name: Start Server
+          command: npm start
+          background: true
```
Comment (zachary-kuhn) [inline .circleci/config.yml]: Oh nice.

File: package.json [hunk 2]
```diff
@@ -1,24 +1,50 @@
 {
   "name": "code-gov-web",
   "version": "1.0.2",
-  "description": "Code.gov is a website promoting good practices in code development, collaboration, and reuse across the U.S.  Government. Code.gov will provide tools and guidance to help agencies implement the [Federal Source Code Policy](https://sourcecode.cio.gov). It will include an inventory of the government's custom code to promote reuse between agencies and will provide tools to help government and the public collaborate on open source projects.",
+  "description": "Code.gov is a website promoting good practices in code development, collaboration, and reuse across the U.S. Government. Code.gov will provide tools and guidance to help agencies implement the [Federal Source Code Policy](https://sourcecode.cio.gov). It will include an inventory of the government's custom code to promote reuse between agencies and will provide tools to help government and the public collaborate on open source projects.",
```
Comment (zachary-kuhn) [inline package.json]: Sweet updates

File: package.json [hunk 3]
```diff
@@ -1,24 +1,50 @@
...
+      "name": "Olivier Kamanda",
+      "url": "https://github.com/okamanda"
+    },
+    {
+      "name": "Luke Keller",
+      "url": "https://github.com/lukad03"
+    },
+    {
+      "name": "Matt Bailey",
+      "url": "https://github.com/mattbaiely0"
```
Comment (zachary-kuhn) [inline package.json]: typo in the URL!

File: package.json (hunk 2 above)
Comment (froi) [reply package.json]: I'm going to miss these sarcastic reviews 😄

File: .circleci/config.yml (hunk 1 above)
Comment (froi) [reply .circleci/config.yml]: Yeah! That really made my day.

**Pull request merged**